js/bootstrap-datepicker.js

```diff
@@ -1545,6 +1545,10 @@
 				p.setRange(range);
 			});
 		},
+		clearDates: function(){
+				this.pickers[0].clearDates();
+				this.pickers[1].clearDates();
```

Comment:
I believe this to be an issue, but maybe it isn't. When calling this `changeDate` also gets called 6 times along with another `clearDate`.  To me the date isn't changing per se, it's being removed.
![image](https://user-images.githubusercontent.com/1037197/30178442-ef2780c4-93d6-11e7-86f8-a5a980d06d2a.png)